src/SamplesApp/UITests.Shared/Windows_Devices/RadioTests.xaml.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using Uno.UI.Samples.Controls;
+using Uno.UI.Samples.UITests.Helpers;
+using Windows.UI.Core;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.Devices.Radios;
+using System.Collections.Generic;
+
+namespace UITests.Shared.Windows_Devices
+{
+	[SampleControlInfo("Windows.Devices", "RadioTests", description: "Demonstrates use of Windows.Devices.Radio.GetRadiosAsync")]
+	public sealed partial class RadioTests : UserControl
+	{
+		public RadioTests()
+		{
+			this.InitializeComponent();
+		}
+
+		private async void uiCreate_Click(object sender, RoutedEventArgs e)
+		{
+			uiResultMsg.Text = "Calling GetRadiosAsync...";
+			try
+			{
+				var radiosList = await Windows.Devices.Radios.Radio.GetRadiosAsync();
+
+				uiResultMsg.Text = "Your current Radios:";
+
+				var intermediateRadiosList = new List<IntermediateRadio>();
+
+				foreach (var oneRadio in radiosList)
+                {

```

Comment:
make sure to use the formatting tools from VS for indentation, same for the associated XAML file.